test/api_callable.js

```diff
@@ -405,6 +403,40 @@ describe('retryable', function() {
     });
   });
 
+  it('errors on maxRetries', function(done) {
```

Comment:
Add comment here about maxRetries being unsupported, and for internal use only. Tests are a form of documentation.